acquia/http-hmac-javascript
# PIN-16014 | Updated crypto-js package version

**Motivation**
Fix vulnerabilities in crypto-js package

**Proposed changes**
Update version to latest = 4.2.0

**Testing steps**
Tested on https://github.com/acquia/acquia-search-rage/pull/2539/commits/fec15fb789c23e5f66754d56ee46c9ac2388d2c2

Local controller
```
[acquia-search-dev]➜  acquia-search-rage git:(PIN-16014) ✗ node controller/main.js  <aws:acquia-search-dev> (node:89377) [DEP0111] DeprecationWarning: Access to process.binding('http_parser') is deprecated.
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:89377) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
redis://localhost:6379 is not a cluster. Connected in standalone server mode
Tue, 26 Dec 2023 10:26:06 GMT [info] GET /v2/stack/get?stack_name=rujuta-v8-stack&region=us-east-1
Tue, 26 Dec 2023 10:30:09 GMT [info] GET /v2/index/get?index_name=RUJU-12345.dev.default
Tue, 26 Dec 2023 10:30:27 GMT [info] GET /v2/index/list?stack_name=rujuta-v8-stack
Tue, 26 Dec 2023 10:31:02 GMT [info] GET /v2/index/ping?index_name=RUJU-88888.36dev.knoqqb42726
Tue, 26 Dec 2023 10:36:00 GMT [info] POST /v2/configset/upload?configset_name=chineseAnalyzerTest&stack_name=rujuta-v8-stack&network_id=RUJU-88888&region=us-east-1
```
search cli cmds are working
